README.md

```diff
@@ -114,49 +114,105 @@ The Vonage Video API Reference App for React is currently supported on the lates
 
 ## Running Locally
 
-### Config
+### 1. Ensure You Have a Vonage Account
 
- In project directory, create the environment variables for the project.
+To get started, make sure you have a Vonage account. You can create one at the [Vonage API Dashboard](https://dashboard.vonage.com/applications). The sign-up process is straightforward.
```

Comment:
I think we can skip on mentioning the following: `The sign-up process is straightforward.`